### ISSUE TITLE
Tarball from release 2.2.0 RC seems to be not well built

### ISSUE THREAD

**Issue by davitol** [owncloud/richdocuments#282]
RichDocuments tarball from 2.2.0.RC tarball contains files like Makefile. Seems to be not well built https://github.com/owncloud/richdocuments/releases/download/2.2.0rc/richdocuments.tar.gz

(cc @timar JFYI)

**Comment by timar** [comment]
Indeed, the tarball contains `a.sh` which was an untracked file in my local build tree. I did not find anything else. Did you? I removed the `a.sh` file, so next build will be OK.

**Comment by davitol** [comment]
> Indeed, the tarball contains `a.sh` which was an untracked file in my local build tree. I did not find anything else. Did you? I removed the `a.sh` file, so next build will be OK

No, anything else. I will close the ticket as soon as next build will be published. Thanks for taking care.

**Comment by davitol** [comment]
> Indeed, the tarball contains `a.sh` which was an untracked file in my local build tree. I did not find anything else. Did you? I removed the `a.sh` file, so next build will be OK.

@timar a.sh is still included in RC2 tarball. Also I guess hidden files from drone should be deleted too?

![Screen Shot 2019-10-31 at 08 31 34](https://user-images.githubusercontent.com/10596560/67927494-0866ff00-fbb9-11e9-8847-f142f4fed7cd.png)
